gobblin-utility/src/main/java/gobblin/util/limiter/DefaultLimiterFactory.java

```diff
@@ -27,13 +27,28 @@
 public class DefaultLimiterFactory {
 
   public static final String EXTRACT_LIMIT_TYPE_KEY = "extract.limit.type";
-  public static final String EXTRACT_LIMIT_RATE_LIMIT_KEY = "extract.limit.rate.limit";
-  public static final String EXTRACT_LIMIT_RATE_LIMIT_TIMEUNIT_KEY = "extract.limit.rate.limit.timeunit";
-  public static final String EXTRACT_LIMIT_TIME_LIMIT_KEY = "extract.limit.time.limit";
-  public static final String EXTRACT_LIMIT_TIME_LIMIT_TIMEUNIT_KEY = "extract.limit.time.limit.timeunit";
+
+  public static final String EXTRACT_LIMIT_RATE_LIMIT_KEY = "extract.limit.rateLimit";
```

Comment:
Do all these constants need to be `public`? If not make them `private` or package protected for unit tests only.